src/Scanner.js

```diff
@@ -195,10 +217,13 @@ class DefaultScanner {
      * @param {HTMLInputElement|HTMLSelectElement} input
      */
     addInput (input) {
+        if (this.stopped) return
```

Comment:
This is needed because we have a `forEach` loop upstream, and even when calling `stopScanner` the loop would continue. This ensures no items are added. Existing snapshot tests catch regressions, so we're good 💪.